src/mapper_node.cpp

```diff
@@ -32,6 +32,14 @@ PM::TransformationParameters robotPoseToSet;
 PM::TransformationParameters previousRobotToMap;
 ros::Time previousTimeStamp;
 
+std::string appendToFilePath(const std::string& filePath, const std::string& suffix)
+{
+	std::string::size_type const ext_pos(filePath.find_last_of('.'));
```

Comment:
For consistency, I would use camelCase for the variable name. Also, please use full words (e.g., `extensionPosition`).